sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
@johnbivens actually it was me who suggested the --no-as-needed flag to @crozeg . I think link time optimization should have removed the library from our binaries too but for some odd reason, it didn't. Probably gcc/clang changed the default behavior across versions? ( https://stackoverflow.com/questions/11631161/force-to-link-against-unused-shared-library )

An alternative would be to create an actual qml plugin out of this as described here ( https://github.com/oKcerG/SortFilterProxyModel/issues/25 ), and then have something like:

```
engine.addPluginPath(QCoreApplication::applicationDirPath());
```